hack/stabilization-changes.py

```diff
@@ -212,6 +263,46 @@ def public_errata_uri(version, cache=None, **kwargs):
     return errata_uri, public
 
 
+def get_concerns_about_risk_extensions(version, channel, update_risks=None):
+    if update_risks is None:
+        return
+
+    release_major_minor = '.'.join(version.split('.', 2)[:2])
+    previous_version = None
+    for target in channel.get('versions', []):
+        target_major_minor = '.'.join(target.split('.', 2)[:2])
+        if target_major_minor != release_major_minor:
+            continue  # we're only looking at earlier patch releases in the same 4.y
+        if sem_ver_less_than(target, version) and (previous_version is None or sem_ver_less_than(previous_version, target)):
+            previous_version = target
+    if previous_version is None:
+        return
+
+    risks = set()
+    previous_risks = {}
+    for path, risk in sorted(update_risks.items()):
+        key = risk.get('name', path)
+        if risk['to'] == version:
+            risks.add(key)
+        if risk['to'] == previous_version:
+            fixed_in = risk.get('fixedIn', None)
+            if fixed_in is None or (version != fixed_in and sem_ver_less_than(version, fixed_in)):
+                previous_risks[key] = fixed_in
+
+    unfixed = []
+    for key, fixed_in in sorted(previous_risks.items()):
+        if key in risks:
+            continue
+        if fixed_in:
+            unfixed.append('{} affects {} and is not fixed until {}.  Extend the risk to {}.'.format(key, previous_version, fixed_in, version))
+        else:
+            unfixed.append('{} affects {}.  Either declare a fix version or extend the risk to {}.'.format(key, previous_version, version))
+
+    if unfixed:
+        return ' '.join(unfixed)
+    return
```

Comment:
nit: you can just unconditionally `return ' '.join(unfixed)`:

```console
>>> ' '.join([])
''
```

Reply:
but I want `None` instead of an empty string, because I like the consistency with the other bare-`return`s.  The consumer is `if concerns_about_risk_extensions`, so your unconditional `join` would work, but it would mean that this function  returns two different values for the different "I don't have any concerns" cases.

Reply:
:grimacing: I dont review too much Python code recently and was reading this as I would read go. All clear now, sorry for the noise.